src/algebra/order/monoid_lemmas_zero_lt.lean

```diff
@@ -212,81 +212,81 @@ variables [preorder α]
 
 -- proven with `a0 : 0 ≤ a` `d0 : 0 ≤ d` as `mul_le_mul_of_le_of_le`
 lemma preorder.mul_le_mul_of_le_of_le [pos_mul_mono α] [mul_pos_mono α]
-  (h₁ : a ≤ b) (h₂ : c ≤ d) (a0 : 0 < a) (d0 : 0 < d) : a * c ≤ b * d :=
-(mul_le_mul_left' h₂ a0).trans (mul_le_mul_right' h₁ d0)
+  (a0 : 0 < a) (d0 : 0 < d) (h₁ : a ≤ b) (h₂ : c ≤ d) : a * c ≤ b * d :=
```

Comment:
```suggestion
  (a0 : 0 < a) (h₁ : a ≤ b) (h₂ : c ≤ d) (d0 : 0 < d) : a * c ≤ b * d :=
```
In cases like this, I wonder whether the "lexicographic" ordering of variables might be preferable.  I am not convinced, it is just a suggestion and I wanted to bring it up, in case someone has a strong preference.